src/govuk/components/exit-this-page/exit-this-page.mjs

```diff
@@ -0,0 +1,343 @@
+/* eslint-disable es-x/no-function-prototype-bind -- Polyfill imported */
+
+import { normaliseDataset } from '../../common/normalise-dataset.mjs'
+import { nodeListForEach, mergeConfigs, extractConfigByNamespace } from '../../common.mjs'
+import { I18n } from '../../i18n.mjs'
+import '../../vendor/polyfills/Element/prototype/classList.mjs'
+import '../../vendor/polyfills/Element/prototype/dataset.mjs'
+import '../../vendor/polyfills/Function/prototype/bind.mjs'
+
+/**
+ * @constant
+ * @type {ExitThisPageTranslations}
+ * @see Default value for {@link ExitThisPageConfig.i18n}
+ * @default
+ */
+var EXIT_THIS_PAGE_TRANSLATIONS = {
+  activated: 'Loading.',
+  timedOut: 'Exit this page expired.',
```

Comment:
Suggestion – would it make sense for these translation key to be more aligned with their English defaults?

```suggestion
  loading: 'Loading.',
  expired: 'Exit this page expired.',
```

Reply:
Good idea. This, as with a few things in the ETP code, are likely hangovers for when the content we included was something else.

Reply:
On a second look I'm going to object to these changes as they most accurately represent what the text is for. "Loading", IMO, wouldn't communicate to folks using the translation API that this is for when the user activates the button. Expired I could be swayed on. Unless you have strong opinions I'm gonna skip over this one.